source/install/build_lammps.sh

```diff
@@ -6,6 +6,14 @@ if [ -z "$FLOAT_PREC" ]
 then
   FLOAT_PREC=high
 fi
+
+if [ ${FLOAT_PREC} == "high" ]; then
+    PREC_DEF="-DHIGH_PREC"
+    PREC_SUFFIX=""
+else
+    PREC_DEF="-DLOW_PREC"
+    PREC_SUFFIX="_low"
```

Comment:
What's the specific name do you mean you need to change? `_low` is used in 3 names, including `-ldeepmd_cc_low`, `USER-DEEPMD_low`, and `ipi_low`.

Reply:
I mean USER-DEEPMD_low.
Otherwise, the following error occurs

`[ludh@login09 src]$ make yes-user-deepmd_Low  `
`Package USER-DEEPMD_LOW does not exist `
`make: *** [Makefile:364: yes-user-deepmd_Low] Error 1
`
I have to do this:
`mv USER-DEEPMD_low USER-DEEPMD_LOW`
then
`make yes-user-deepmd_low`
`Installing package user-deepmd_low`